packages/nelua/Makefile.patch

```diff
@@ -1,15 +1,14 @@
-diff -uNr nelua-lang/Makefile nelua-lang.mod/Makefile
---- nelua-lang/Makefile	2022-08-28 12:14:53.076791242 +0800
-+++ nelua-lang.mod/Makefile	2022-08-28 12:16:10.306791213 +0800
+--- ./Makefile	2024-04-16 20:58:39.980928203 +0800
++++ ./Makefile	2024-04-16 21:01:18.432928142 +0800
 @@ -40,7 +40,6 @@
- OPT_CFLAGS=-O3 -flto -fno-plt -fno-stack-protector
+ FAST_CFLAGS=-O3 -flto=auto -fno-plt -fno-stack-protector
+ ARCH_CFLAGS=-march=native
  ifeq ($(SYS), Linux)
- 	CFLAGS=-std=gnu99 -O2
 -	CC=gcc
- 	DEFS+=-DLUA_USE_LINUX
- 	LIBS+=-lm -ldl
- 	LDFLAGS+=-Wl,-E
-@@ -227,7 +226,7 @@
+ 	DEFS+=-std=gnu99 -DLUA_USE_LINUX
+ 	LIBS+=-lm -ldl -Wl,-E
+ 	ifeq ($(CC), musl-gcc)
```

Comment:
Hmm, looks fine from the logs...
https://github.com/termux/termux-packages/actions/runs/8752108049/job/24019077648?pr=19789#step:8:479

Reply:
This is a patch file. It may contains lines of from the original file to be patched. Generally I review patch file as a whole (https://github.com/termux/termux-packages/blob/427c404a2b445cc99868bfcfc7276db234e23fcd/packages/nelua/Makefile.patch) instead of a diff on a patch.

Reply:
Mmh yeah fair enough.
This just felt like the appropriate place to bring it up.